agent/Cargo.toml

```diff
@@ -14,46 +14,48 @@ akri-opcua = { path = "../discovery-handlers/opcua", optional = true }
 akri-udev = { path = "../discovery-handlers/udev", optional = true }
 akri-shared = { path = "../shared" }
 anyhow = "1.0.38"
-async-stream = "0.2"
+async-stream = "0.3"
 async-trait = "0.1.0"
 blake2 = "0.9.0"
 chrono = "0.4.10"
 cfg-if = "1.0.0"
-env_logger = "0.8.3"
+env_logger = "0.9.0"
 futures = { version = "0.3.1", package = "futures" }
 futures-core = "0.3"
 futures-util = "0.3"
-hyper = "0.13.10"
-h2 = "=0.2.6"
-kube = { version = "0.23.0", features = ["openapi"] }
-k8s-openapi = { version = "0.6.0", features = ["v1_16"] }
+hyper = "0.14.2"
+h2 = "0.3.0"
```

Comment:
Lock h2 version so that it won't update when doing cardo update (ie: "=0.3.0")

Reply:
Thank you! fixed in d4bab9a